tests/testthat/_snaps/snapshot.md

```diff
@@ -243,11 +243,11 @@
     Code
       cat(snapshot_accept_hint("_default", "bar.R"))
     Output
-      * Run `snapshot_accept('bar.R')` to accept the change
-      * Run `snapshot_review('bar.R')` to interactively review the change
+      * Run `]8;;rstudio:run:testthat::snapshot_accept('bar.R')snapshot_accept('bar.R')]8;;` to accept the change
+      * Run `]8;;rstudio:run:testthat::snapshot_review('bar.R')snapshot_review('bar.R')]8;;` to interactively review the change
```

Comment:
cli is not using the ideal ansi sequence for links currently, because rstudio does not support that yet. But once we fix this in rstudio, cli will switch to that as well, so these snapshots will change. fyi.

Reply:
Ok, sounds good.